css/navigation.css

```diff
@@ -1,26 +1,31 @@
+:root {
+	--ion-surface-primary: var(--ion-color-main-background);
+	--ion-surface-secondary:  var(--ion-color-cool-grey-c1);
```

Comment:
To me "surface" feels a little unclear... maybe something like "overlay-background" or "layer-background" describes a little bit better what it's actually about? :thinking: